src/Traits/HasRoles.php

```diff
@@ -167,6 +168,32 @@ public function hasPermissionTo($permission)
     }
 
     /**
+     * Determine if the user has any of the given permission(s).
+     *
+     * @param string|array|Permission|\Illuminate\Support\Collection $permissions
+     *
+     * @return bool
+     */
+    public function hasAnyPermission($permissions)
+    {
+        if (is_string($permissions)) {
+            try {
+                return $this->hasPermissionTo($permissions);
+            } catch (PermissionDoesNotExist $e) {
```

Comment:
This is the one line I'm not certain you want in here since your `hasPermissionTo` method doesn't attempt to catch any missing permissions. Would love to hear the feedback and am available to update the PR as necessary.